### ISSUE TITLE
BSoftBody editor not saving mesh data?

### ISSUE THREAD

**Issue by iakat** [Phong13/BulletSharpUnity3d#76]
Hi, I am having this issue but I am not sure whether I am doing something wrong or not.

I am running the git version of BulletSharp, and I even tried #54 with no luck.

BSoftBody editor for me does not save the mesh data which results in not being able to edit+save any settings, see [this video](https://simone.sh/images/2009221609_mxuTUNcQBZ1EZvrxWG.mp4) for an example

Even getting all settings right and then trying to save it, I get this error when play testing, where Wheel 1 is my BSoftBody.

```
Could not build mesh from meshSettings for Wheel 1 (BulletUnity.BSoftBodyWMesh)
UnityEngine.Debug:LogError(Object)
BulletUnity.BSoftBodyWMesh:_BuildCollisionObject() (at Assets/BulletUnity/Scripts/SoftBody/BSoftBodyWMesh.cs:33)
BulletUnity.BCollisionObject:GetCollisionObject() (at Assets/BulletUnity/Scripts/BCollisionObject.cs:168)
BulletUnity.BSoftBodyEditor:OnSceneGUI() (at Assets/BulletUnity/Scripts/Editor/BSoftBodyEditor.cs:239)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

```

Any tips would be appreciated. Thanks!

**Comment by iakat** [comment]
I think this happens to a lesser degree to the RigidBody editor as well, where the mesh renderer does not get the mesh data

**Comment by Phong13** [comment]
Will try to take a look later today. Are you using Unity 2019?

**Comment by iakat** [comment]
Yes, 2019.4 (full version on the top left in the video linked). Thanks a  lot!

**Comment by Phong13** [comment]
I am able to reproduce the problem. Hope to push a solution in a few days.
